flaml/integrations/oai/completion.py

```diff
@@ -187,34 +188,40 @@ def eval(cls, config: dict, prune=True, eval_only=False):
         prune_hp = cls._prune_hp
         metric = cls._metric
         config_n = config[prune_hp]
-        max_tokens = config["max_tokens"]
+        max_tokens = config.get("max_tokens", 16)  # default value is 16
```

Comment:
why 16? is there any reference for how 16 is selected?

Reply:
default value in OpenAI.

Reply:
Okay. Shall we mention it's the default value in OpenAI?